src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -141,6 +141,8 @@ public void Initialize()
 
             this.StartNodeServer();
 
+            this.selfEndpointTracker.UpdateAndAssignMyExternalAddress(null, this);
```

Comment:
Can the parameter you are supplying as null be an optional parameter in UpdateAndAssignMyExternalAddress ?